obspy/signal/spectral_estimation.py

```diff
@@ -53,6 +54,90 @@
 NOISE_MODEL_FILE = os.path.join(os.path.dirname(__file__),
                                 "data", "noise_models.npz")
 
+earthquakes = {
+    (1.5,10): [[7.0700000e-01, 1.4140000e+00, 2.8280000e+00, 5.6600000e+00, 
+                1.1300000e+01, 2.2600000e+01],
+               [1.1940722e-06, 5.9652811e-06, 4.2022150e-05, 2.1857010e-04, 
+                6.4011669e-04, 1.0630361e-03]],
```

Comment:
I think we could live with some less digits in the data variables? `%.2e` should be enough maybe?

Reply:
On principle yes but highly prone to small error I rather not if allowed.

Reply:
Ok, if you prefer then leave it as is.
(I don't think it would make any difference though, especially since it is a log-log plot)